static/app/components/compactSelect/gridList/option.tsx

```diff
@@ -42,7 +43,7 @@ export function GridListOption({node, listState, size}: GridListOptionProps) {
     ? selectionMode === 'multiple'
     : listState.selectionManager.selectionMode === 'multiple';
 
-  const {rowProps, gridCellProps, isSelected, isDisabled} = useGridListItem(
+  const {rowProps, gridCellProps, isSelected, isDisabled, isPressed} = useGridListItem(
```

Comment:
Using `react-aria`'s `isPressed` prop to track press states (the value is passed to `InteractionStateLayer`). This is better because the `:active` pseudo-class is activated even when the user presses on a child interactive element.

**Before ——** pressing on the bookmark star activates the whole option's press state, which is confusing because it suggests that the option will be selected (it won't)
![image](https://user-images.githubusercontent.com/44172267/228977520-1835a5cf-ad75-4ec9-a01b-1f8410505242.png)

**After ——** pressing the bookmark star doesn't trigger the option's press state
![image](https://user-images.githubusercontent.com/44172267/228977180-0d30e25d-3dfc-4a7f-a006-79be7e1857d1.png)